Extensions/blacklist.js

```diff
@@ -152,6 +156,9 @@ XKit.extensions.blacklist = new Object({
 
 		$(document).on('click', ".xblacklist_open_post", XKit.extensions.blacklist.unhide_post);
 
+		const postSel = XKit.css_map.keyToCss('listTimelineObject') || '.post';
+		const postContentSel = XKit.css_map.keyToCss('post') || '.post_content';
```

Comment:
because "post" refers to many different things (4 here, to be precise), this constant ends up being `._2OqKn, ._2-mU3, ._1yQrs, ._1Wz4U`